webclient/app/src/app/commons/alert/ConfirmationDialog.jsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import { useTranslation } from 'react-i18next';
+
+export default function ConfirmationDialog(props) {
+    const { content, open, onClose, onSubmit } = props;
+    const { t } = useTranslation();
+
+    return (
+        <Dialog
+            open={open}
+            onClose={onClose}
+            aria-labelledby="alert-dialog-title"
+            aria-describedby="alert-dialog-description"
+        >
+            <Alert severity="warning">
+                <AlertTitle>{t(content.title)}</AlertTitle>
+                {t(content.message)}
```

Comment:
Please move the translation part to the parent element, so that a pure string will be displayed here
```suggestion
                {content.message}
```